### DISCUSSION TITLE
What makes path-loss considerations give me a higher BER?

### DISCUSSION THREAD

**Discussion by anindha-dhar** [NVlabs/sionna#333] [Q&A] [answered]
If I don't consider **path loss (i.e Normalized )** , I get a _very low BER and average power =  1_
![WhatsApp Image 2024-02-04 at 23 00 48_98bf1b4a](https://github.com/NVlabs/sionna/assets/66663552/a77fed8e-1a6c-4c62-a8c2-10c78b7fc05b)
![WhatsApp Image 2024-02-04 at 23 00 08_6e6ce55c](https://github.com/NVlabs/sionna/assets/66663552/f033051a-3fdc-421d-a147-423ddb8f3e60)

But if I **consider path loss**, I will always get a _BER almost 50%_
![WhatsApp Image 2024-02-04 at 23 02 08_fc545e58](https://github.com/NVlabs/sionna/assets/66663552/3126f34c-e0b1-4d50-b7f0-159fc0b7d6c0)

I tuned the parameter **_normalized = False_** as mentioned in the comment:

![image](https://github.com/NVlabs/sionna/assets/66663552/c7b82714-ee3c-4cb8-ba0e-a45d65c6ca5a)

What is the main reason behind this?

**Comment by SebastianCa** [comment]
Hi,
the reason for the observed behavior is (most likely) that the enabled path loss causes an attenuation of the tx signal. You could verify this by testing the average power of `h` with enabled path loss. You need to adjust the noise power (i.e., `no`) accordingly. Otherwise, a different SNR point is simulated which explains the different BER.

**Reply by anindha-dhar** [reply]
Thanks for the reply.
Based on your suggestion, I have adjusted the noise power (i.e., `no`) accordingly to mitigate the effects of signal degradation. However, despite these efforts, I continue to encounter a significant 50% Bit Error Rate (BER).

Can you please suggest me some more ways to reduce the BER considering the path-loss? Maybe I have made some mistakes while working on your suggestion. Perhaps an example can clear my doubt.

Thanks in advance. I appreciate your time and effort.

**Reply by SebastianCa** [reply]
It is difficult to give advice without knowing the exact system setup. Could you please provide a [gist](https://gist.github.com/) that reproduces your observed behavior?

**Reply by anindha-dhar** [reply]
I am actually running the exact same code of [sionna ray tracing notebook](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html)

But while calculating the BER, I just tuned the parameter `normalize = False` to observe the path-loss behavior. But considering path loss giving me higher BER near about 50%. I need to find a way or solution to reduce the BER.

Thanks in advance.

**Reply by jhoydis** [reply, marked as answer]
Hi, 

In this notebook, users are located at different positions within the scene. There are hence very large path loss differences between the positions. 

In order to make meaningful BER simulations, one needs to be able to define an SNR. We have assumed that there is perfect power control in place that compensates for the path loss. That is achieved by setting ``normalize=True``, i.e., all users see channels with a path loss equal to 0dB.

If you now disable the normalization, you can have one user with an 80dB path loss and one user with a 130dB path loss and it will be essentially impossible to decode both correctly as they have a 50dB difference in received signal strength.

There is hence no way to reduce the BER or make it work.

What are you trying to show with your simulations?

**Reply by anindha-dhar** [reply]
Basically I'm trying to build a digital twin, which mimics a practical cellular network. So I was considering path-loss, but it's giving me a high BER. So I was wondering if there is any way to reduce this.
